internal/subscription/model.go

```diff
@@ -19,6 +20,9 @@ const (
 
 	// SubscriptionTypePull ...
 	SubscriptionTypePull = "Pull"
+
+	// DeadLetterTopicSuffix dead letter topic suffix
+	DeadLetterTopicSuffix = "-dlq"
```

Comment:
why not reuse DeadLetterTopicSuffix defined in internal/topic package..?

Reply:
used topic suffix nmow